<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup_page</class>
 <widget class="QMainWindow" name="Signup_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget
{
	background:url(:/source/back_1.jpg);
}
*{
font: 9pt &quot;Bauhaus 93&quot;;
font-size:24px;
}
QToolButton{
background:pink;
border-radius:60px;
}
QPushButton
{
background:white;
border-radius:15px;
font: 12pt &quot;华文琥珀&quot;;
}
QPushButton:hover
{
background:pink;
border-radius:15px;
font: 12pt &quot;华文琥珀&quot;;
}
QLineEdit
{
background:rgba(255,255,255,0.5);
border:none;
color:white;
font: 12pt &quot;华文琥珀&quot;;
background:transparent;
border-bottom:1px solid #717072;
}
QFrame{
background:rgba(255,255,255,0.6);
border-radius:15px;
}
QLabel
{
background:transparent;
color:black;
font:20pt &quot;华文琥珀&quot;;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_question">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>设置密保问题</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_answer">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>密保答案</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>570</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="placeholderText">
     <string>电子邮箱</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_signup_now">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>670</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>立刻注册</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
